### PR TITLE
Fix highcharts

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: templates/frame.haml [hunk 1]
```diff
@@ -80,6 +80,7 @@
         var static_url = '{{STATIC_URL}}';
 
       %script{src:"{{ STATIC_URL }}js/libs/jquery.url.js"}
+      %script{src:"{{ STATIC_URL }}bower/bootstrap/js/bootstrap-tab.js"}
```
Comment (ericnewcomer) [inline templates/frame.haml:83]: What is this one for? Old flow results? I'd prefer to replace with temba-tabs as the fix wherever this is.

**Review by ericnewcomer — CHANGES_REQUESTED** [review]

File: templates/frame.haml (hunk 1 above)
Comment (norkans7) [reply templates/frame.haml:83]: Yes, that is for `.tab-pane` on https://github.com/nyaruka/rapidpro/blob/2c2cd31f5aab756d85a3b0e95da90c4eda40089b/templates/flows/flow_results.haml#L626

**Comment by norkans7** [comment]
Monosnap RapidPro - Visually build nationally scalable mobile applications 2022-12-15 18-04-10

**Comment by norkans7** [comment]
replaced by #4202

**Pull request closed without merging**